### ISSUE TITLE
[Bug] 部分项目5月份的rank数据缺失

### ISSUE THREAD

**Issue by Mjennie** [X-lab2017/open-digger#1786]
## 问题描述
以下项目所在表格表示改项目截止目前获取到的最新月份的rank数据，有21个项目4月份还能获取到rank数据，但是5月份就无法获取。请查看是否可以修复这些项目的每月rank数据，望回复。
（若存在不活跃项目不再获取rank统计数据，是否可以对此类项目进行区分）

### 2026-04

| 项目 | 最新月份 | 链接 |
|------|----------|------|
| NVlabs/Fast-dLLM | 2026-04 | [链接](https://oss.open-digger.cn/github/NVlabs/Fast-dLLM/openrank.json) |
| hpcaitech/ColossalAI | 2026-04 | [链接](https://oss.open-digger.cn/github/hpcaitech/ColossalAI/openrank.json) |
| huggingface/pytorch-image-models | 2026-04 | [链接](https://oss.open-digger.cn/github/huggingface/pytorch-image-models/openrank.json) |
| inclusionAI/dInfer | 2026-04 | [链接](https://oss.open-digger.cn/github/inclusionAI/dInfer/openrank.json) |
| michaelfeil/infinity | 2026-04 | [链接](https://oss.open-digger.cn/github/michaelfeil/infinity/openrank.json) |
| ByteDance-Seed/Triton-distributed | 2026-04 | [链接](https://oss.open-digger.cn/github/ByteDance-Seed/Triton-distributed/openrank.json) |
| HazyResearch/ThunderKittens | 2026-04 | [链接](https://oss.open-digger.cn/github/HazyResearch/ThunderKittens/openrank.json) |
| facebookresearch/CUTracer | 2026-04 | [链接](https://oss.open-digger.cn/github/facebookresearch/CUTracer/openrank.json) |
| Lightning-AI/litgpt | 2026-04 | [链接](https://oss.open-digger.cn/github/Lightning-AI/litgpt/openrank.json) |
| adapter-hub/adapters | 2026-04 | [链接](https://oss.open-digger.cn/github/adapter-hub/adapters/openrank.json) |
| huggingface/open-r1 | 2026-04 | [链接](https://oss.open-digger.cn/github/huggingface/open-r1/openrank.json) |
| inclusionAI/AReaL | 2026-04 | [链接](https://oss.open-digger.cn/github/inclusionAI/AReaL/openrank.json) |
| meta-pytorch/OpenEnv | 2026-04 | [链接](https://oss.open-digger.cn/github/meta-pytorch/OpenEnv/openrank.json) |
| thu-ml/tianshou | 2026-04 | [链接](https://oss.open-digger.cn/github/thu-ml/tianshou/openrank.json) |
| NVIDIA-Merlin/Transformers4Rec | 2026-04 | [链接](https://oss.open-digger.cn/github/NVIDIA-Merlin/Transformers4Rec/openrank.json) |
| open-mmlab/mmdetection | 2026-04 | [链接](https://oss.open-digger.cn/github/open-mmlab/mmdetection/openrank.json) |
| open-mmlab/mmdetection3d | 2026-04 | [链接](https://oss.open-digger.cn/github/open-mmlab/mmdetection3d/openrank.json) |
| deepseek-ai/DeepSeek-V3.2-Exp| 2026-04| [链接](https://oss.open-digger.cn/github/deepseek-ai/DeepSeek-V3.2-Exp/openrank.json) |
| open-compass/opencompass | 2026-04 | [链接](https://oss.open-digger.cn/github/open-compass/opencompass/openrank.json) |
| huggingface/pytorch-image-models | 2026-04 | [链接](https://oss.open-digger.cn/github/huggingface/pytorch-image-models/openrank.json) |
| openai/whisper | 2026-04 | [链接](https://oss.open-digger.cn/github/openai/whisper/openrank.json) |

### 2026-03

| 项目 | 最新月份 | 链接 |
|------|----------|------|
| huggingface/text-generation-inference | 2026-03 | [链接](https://oss.open-digger.cn/github/huggingface/text-generation-inference/openrank.json) |
| sgl-project/ome | 2026-03 | [链接](https://oss.open-digger.cn/github/sgl-project/ome/openrank.json) |
| mistralai/mistral-finetune | 2026-03 | [链接](https://oss.open-digger.cn/github/mistralai/mistral-finetune/openrank.json) |
| stochasticai/xTuring | 2026-03 | [链接](https://oss.open-digger.cn/github/stochasticai/xTuring/openrank.json) |
| deepseek-ai/FlashMLA | 2026-03 | [链接](https://oss.open-digger.cn/github/deepseek-ai/FlashMLA/openrank.json) |
| meta-pytorch/torchforge | 2026-03 | [链接](https://oss.open-digger.cn/github/meta-pytorch/torchforge/openrank.json) |
| opea-project/GenAIInfra | 2026-03 | [链接](https://oss.open-digger.cn/github/opea-project/GenAIInfra/openrank.json) |

### 2026-02

| 项目 | 最新月份 | 链接 |
|------|----------|------|
| Vchitect/VBench | 2026-02 | [链接](https://oss.open-digger.cn/github/Vchitect/VBench/openrank.json) |
| aigw-project/aigw  | 2026-02 | [链接](https://oss.open-digger.cn/github/aigw-project/aigw/openrank.json) |
| zhihu/ZhiLight | 2026-02 | [链接](https://oss.open-digger.cn/github/zhihu/ZhiLight/openrank.json) |
| microsoft/triton-shared | 2026-02 | [链接](https://oss.open-digger.cn/github/microsoft/triton-shared/openrank.json) |
| OptimalScale/LMFlow | 2026-02 | [链接](https://oss.open-digger.cn/github/OptimalScale/LMFlow/openrank.json) |
| PaddlePaddle/PaddleNLP | 2026-02 | [链接](https://oss.open-digger.cn/github/PaddlePaddle/PaddleNLP/openrank.json) |
| NVIDIA-Merlin/HierarchicalKV | 2026-02 | [链接](https://oss.open-digger.cn/github/NVIDIA-Merlin/HierarchicalKV/openrank.json) |
| deepseek-ai/smallpond | 2026-02 | [链接](https://oss.open-digger.cn/github/deepseek-ai/smallpond/openrank.json) |
| ModelEngine-Group/fit-framework | 2026-02 | [链接](https://oss.open-digger.cn/github/ModelEngine-Group/fit-framework/openrank.json) |

### 2026-01

| 项目 | 最新月份 | 链接 |
|------|----------|------|
| Tencent/KsanaLLM | 2026-01 | [链接](https://oss.open-digger.cn/github/Tencent/KsanaLLM/openrank.json) |
| AI4Finance-Foundation/ElegantRL | 2026-01 | [链接](https://oss.open-digger.cn/github/AI4Finance-Foundation/ElegantRL/openrank.json) |
| deepseek-ai/DualPipe | 2026-01 | [链接](https://oss.open-digger.cn/github/deepseek-ai/DualPipe/openrank.json) |
| deepseek-ai/EPLB | 2026-01 | [链接](https://oss.open-digger.cn/github/deepseek-ai/EPLB/openrank.json) |
| NVIDIA-Merlin/HugeCTR | 2026-01 | [链接](https://oss.open-digger.cn/github/NVIDIA-Merlin/HugeCTR/openrank.json) |
| alibaba/RecIS | 2026-01 | [链接](https://oss.open-digger.cn/github/alibaba/RecIS/openrank.json) |
| zai-org/GLM-4 | 2026-01 | [链接](https://oss.open-digger.cn/github/zai-org/GLM-4/openrank.json) |

**Comment by frank-zsy** [comment]
由于 OpenDigger 对于当月不活跃的项目，指标不会导出 0 值，而是直接缺失，我没有完全确认上述所有项目，但抽查了其中约 10 个项目，都是 5 月已经完全不活跃的（由于 GitHub 日志出现了缺失，不一定是真正的完全不活跃，但从数据上确实是 5 月没有 Issue 和 PR 活跃的）。

以第一个项目 NVlabs/Fast-dLLM，其在 2、3 月份也没有活跃，因此 OpenRank 数据中，2026-01 后没有 2 和 3 月数据，4 月有活跃后有 2026-04 数据点，后续 5 月没有活跃，因此就没有 2026-05 的数据点，可以认为这个数据点就是 0。

可以使用 HyperCRX 插件查看一下上述所有项目在 5 月的 Issue、PR 活跃情况，也可以简单看是否有参与者和贡献者数量，如果都是 0，表示该项目（在数据上）5 月份是没有活跃的，因此没有导出 2026-05 数据点是符合预期的。如果还有某些仓库是有活跃，但 OpenRank 数值未导出的，请再评论告知。

OpenDigger 不会移除曾经在导出范围内的项目，因此没有取消导出的机制，一般这种情况都是由于项目已经完全不活跃导致的。

**Comment by Mjennie** [comment]
好的，已了解，非常感谢。

**Comment by frank-zsy** [comment]
@Mjennie 可否问一下您是什么情况下使用 OpenDigger 的数据吗？具体的使用场景，也可以定制化输出更高质量的数据。

**Comment by Mjennie** [comment]
我主要用于关注项目每月动态。我关注了一些项目，所以每个月rank数据缺失就能很明显看到变化。不过项目不活跃没有rank数据其实也是符合项目关注预期的，后续会使用类似HyperCRX 插件先判断项目是否活跃。
